libraries/chain/include/eosio/chain/pending_snapshot.hpp

```diff
@@ -50,6 +51,8 @@ class pending_snapshot {
                  "Unable to finalize valid snapshot of block number ${bn}: [code: ${ec}] ${message}",
                  ("bn", get_height())("ec", ec.value())("message", ec.message()));
 
+      ilog("Snapshot created at block ${bn} available as file ${fn}", ("bn", block_ptr->block_num())("fn", fs::path(final_path).filename()));
```

Comment:
Seems would be better to log the full path with filename, not just the filename.